src/vstest.console/Properties/launchSettings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
to remove this one?

Reply:
> It's great, maybe follow-up with docs update

I would wait a bit with it, it's useful mostly for our team, and I was using simplified version of this for a while, but not this one, so maybe there will be some more feedback.